fab/operations/supervisor.py

```diff
@@ -198,7 +199,11 @@ def _format_env(current_env, extra=None):
     ]
 
     host = current_env.get('host_string')
```

Comment:
This confused me, but I confirmed that this function *is* called with `current_env['host_string']` equal to each of hosts.